qface/idl/domain.py

```diff
@@ -435,6 +435,10 @@ def toJson(self):
         o['type'] = self.type.toJson()
         return o
 
+    @property
+    def hasReturnValue(self):
```

Comment:
I would like to avoid to expose type information on the symbol level. How about adding a `is_void` to the type. Than you can write:
```
symbol.type.is_void
```

instead of
```
symbol.hasReturnValue
```